packaging/testing/smoke/packages/Dockerfile.ubuntu2204

```diff
@@ -0,0 +1,30 @@
+# For staging upgrade we use the 'official-install' as the base
+
+FROM dokken/ubuntu-22.04:latest as official-install
```

Comment:
Just add the Hadolint ignores for now to remove these warnings
`# hadolint ignore=DL3007`